renderlib/CMakeLists.txt

```diff
@@ -71,15 +71,15 @@ add_compile_definitions(_STATICLIBBUILD)
 # I prefer target_compile_definitions but the target has to exist first
 add_subdirectory(libCZI/Src/JxrDecode)
 add_subdirectory(libCZI/Src/libCZI)
-find_library(libCZI REQUIRED PATHS ${CMAKE_SOURCE_DIR}/libCZI)
+find_library(libCZI PATHS ${CMAKE_BINARY_DIR}/renderlib/libCZI/Src/libCZI)

```

Comment:
What's the reasoning for removing the REQUIRED keyword?

Reply:
To be honest, that may be an accidental removal for a scenario where it wasn't finding it in the right place.  It's clearly required implicitly anyway, as the build will fail without it.